docs/07. media library/02. integrating in your module.md

```diff
@@ -0,0 +1,282 @@
+ *      onDelete="cascade"
+ * )
+ */
+protected $group;
+```
+
+```php
+/**
+ * Gets the value of group.
+ *
+ * @return MediaGroup
+ */
+public function getGroup()
+{
+    return $this->group;
+}
+```
+
+You will also need to add set `$this->group = $group` in your constructor (and `create`) and `update` method.
+
+
+### In your Symfony FormType class
+
+```php
+use Backend\Modules\MediaLibrary\Domain\MediaGroup\MediaGroupType;
+```
+
+```php
+$builder->add(
+    'mediaGroup',
+    MediaGroupType::class,
+    [
+        'label' => 'lbl.MediaConnected',
+    ]
+)
+```
+> You can replace `lbl.MediaConnected` by your own backend label, f.e.: 'Screenshots', 'Trailers', 'Preview images', ...
+
+Example: `Backend/Modules/MediaGalleries/Domain/MediaGallery/MediaGalleryType.php#L92`
+
+### In your Create Command class
+
+```php
+use Backend\Modules\MediaLibrary\Domain\MediaGroup\MediaGroup;
+use Backend\Modules\MediaLibrary\Domain\MediaGroup\Type as MediaGroupType;
+```
+
+```php
+/**
+ * @var MediaGroup
+ */
+public $mediaGroup;
+```
+
+**In your __construct()**
+```php
+$this->mediaGroup = MediaGroup::create(MediaGroupType::fromString('image'));
+```
+> Possible MediaGroupType values are; "all", "audio", "file", "image", "image-file", "image-movie", "movie". Which defines which MediaItem files are allowed in this MediaGroup.
+
+Example: `src/Backend/Modules/MediaGalleries/Domain/MediaGallery/Command/CreateMediaGallery.php`
+
+Example line: `src/Backend/Modules/MediaGalleries/Domain/MediaGallery/Command/CreateMediaGallery.php#L64`
+
+### In your Create Command Handler class
+
+Don't forget to inject your new MediaGroup into your CustomEntity.
+
+Example line: `src/Backend/Modules/MediaGalleries/Domain/MediaGallery/Command/CreateMediaGalleryHandler.php#L35` 
+
+### In your Add action
+
+```php
+$this->tpl->assign('mediaGroup', $form->getData()->mediaGroup);
+```
+
+Example: `/src/Backend/Modules/MediaGalleries/Actions/Add.php
+
+### In your Update Command class
+
+```php
+use Backend\Modules\MediaLibrary\Domain\MediaGroup\MediaGroup;
+```
+
+```php
+/**
+ * @var MediaGroup
+ */
+public $mediaGroup;
+```
+
+**In your __construct()**
+```php
+$this->mediaGroup = $customEntity->getGroup();
+```
+OR if you are integrating mediaGroup in an existing module, use the following:
+```
+$this->mediaGroup = $customEntity->getGroup();
+
+if (!$this->mediaGroup instanceof MediaGroup) {
+    // Note: I'm using 'image' in this example, use what you want, ...
+    $this->mediaGroup = MediaGroup::create(MediaGroupType::fromString('image'));
+}
+```
+
+Example: `src/Backend/Modules/MediaGalleries/Domain/MediaGallery/Command/UpdateMediaGallery.php#L61`
+
+### In your Edit action
+
+```php
+$this->tpl->assign('mediaGroup', $form->getData()->mediaGroup);
+```
+
+Example: `src/Backend/Modules/MediaGalleries/Actions/Edit.php`
+
+### In your Add and Edit template
+
+This will result in:
+![Screenshow of the Backend Meda Helper](./assets/integration-example-01.png)
```

Comment:
This screenshot is outdated (no Bootstrap) + in the wrong dutch (should be in english)